src/Npgsql/Internal/ResolverFactories/UnmappedTypeInfoResolverFactory.Multirange.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Npgsql.Internal.Converters;
+using Npgsql.Internal.Postgres;
+using Npgsql.PostgresTypes;
+using NpgsqlTypes;
+
+namespace Npgsql.Internal.ResolverFactories;
+
+sealed partial class UnmappedTypeInfoResolverFactory
```

Comment:
nit: I'd just shove them into the same file, IMHO doesn't justify partial here

Reply:
Yeah I suppose you're right. Having done these on autopilot I didn't really register how small these are.